### DISCUSSION TITLE
Exporting weights of models

### DISCUSSION THREAD

**Discussion by paul-reiners** [MIC-DKFZ/nnUNet#1003] [Q&A] [answered]
Is it possible to export the weights of an nnU-Net model so that others can import them?  Something like [Torch Hub](https://pytorch.org/docs/stable/hub.html)?

**Comment by FabianIsensee** [comment, marked as answer]
Weights without the architecture to put them in are rather useless (unless you export as onnx). But sure you can do that. Just read the checkpoint files with torch.load. There is a dictionary in there where you will be able to find the model weights as well
